src/docs/guides/database-migration-guide.md

```diff
@@ -77,6 +77,22 @@ If you would prefer not to go through the migration flow as outlined above, ther
     - [Redis](https://railway.app/template/redis-plugin-migration)
 2. **DIY** - If our tools don't align with your needs, or you have a unique migration process in mind, you're welcome to execute your own strategy.  We always recommend ensuring you have a backup and a process for testing.
 
+## Migration Timeline
+
+- On **January 31st, 2024**:
+  - We will begin a rolling shutdown of all database plugins
+  - We expect all database plugins to be shutdown by **Feburary 3rd, 2024**
+  - After your database plugin is shut down, it will be **inaccessible**, and **data will remain intact**
+  - Impacted users will receive an email with instructions on how to proceed
+- Between **January 31st, 2024** and **Feburary 14th, 2024**:
+  - Users will have a final **2-week window to migrate** their database plugins to the new version
+  - We will begin **archiving data** of database plugins that were shut down as a result of this deprecation
+    - Archived data will be available for download upon request for up to **6 months** from **Feburary 14th, 2024**
+    - If your **database plugin is locked, your data will not be archived**. Please **upgrade to a paying plan** to unlock your database plugin and migrate
```

Comment:
Confusing participle, if I don't reenable my account will my data be lost? Or can I upgrade at any point within the 6 month period to regain my data?

Reply:
Added a "it will be deleted" blurb:

> If your database plugin is locked, your data will not be archived, and it will be deleted on Feburary 14th, 2024. Please upgrade to a paying plan to unlock your database plugin and migrate